adaptived/tests/ftests/1000-sudo-effect-sd_bus_setting_set_int.json

```diff
@@ -10,6 +10,16 @@
 				}
 			],
 			"effects": [
+				{
+					"name": "sd_bus_setting",
```

Comment:
Is this effect needed?  I think `MemoryMax` is set to `infinity` by default.

Or are you using this to test the set and validate logic?

Reply:
I am finding that if the test is run more than once, the last value set is what it starts with not _infinity_. Not sure what is "remembering" the last value.

Reply:
We are modifying MemoryMax. Added this debug to show the issue:
```
diff --git a/adaptived/src/utils/sd_bus_utils.c b/adaptived/src/utils/sd_bus_utils.c
index 1896ff5..c401be9 100644
--- a/adaptived/src/utils/sd_bus_utils.c
+++ b/adaptived/src/utils/sd_bus_utils.c
@@ -368,9 +368,25 @@ API int adaptived_sd_bus_set_ll(const char * const target, const char * const pr
        val.type = ADAPTIVED_CGVAL_LONG_LONG;
        val.value.ll_value = value;
 
+       {
+               long long ll_value2 = 0;
+
+fprintf(stderr, "\nXXX PRE:\n");
+               system("cat /sys/fs/cgroup/sudo1000.slice/memory.max");
+               ret = adaptived_sd_bus_get_ll(target, property, &ll_value2);
+fprintf(stderr, "XXX adaptived_sd_bus_set_ll: First get adaptived_sd_bus_get_ll() target=%s, property=%s, ll_value2=%lld, ret=%d\n",
+    target, property, ll_value2, ret);
+fprintf(stderr, "XXX POST:\n");
+               system("cat /sys/fs/cgroup/sudo1000.slice/memory.max");
+fprintf(stderr, "\n");
+       }
        ret = set_property(target, property, &val);
        if (ret < 0)
                return ret;
+fprintf(stderr, "XXX POST set_property():\n");
+               system("ls /sys/fs/cgroup/sudo1000.slice/memory.max");
+               system("cat /sys/fs/cgroup/sudo1000.slice/memory.max");
+fprintf(stderr, "\n");
 
        if (ret == 0 && (flags & ADAPTIVED_CGROUP_FLAGS_VALIDATE)) {
                long long validate_value;
```
And here's the output:
```
sudo1000 .
Debug: Initializing cause always
Debug: Initializing effect sd_bus_setting
Info: Failed to find key pre_set_from
Debug: Parsed cgroup value: ll_value = 89997312
Info: Failed to find key limit
Debug: sd_bus setting: validate = 1
Debug: adaptived_loop: Debug mode. Skip running as daemon.
Debug: Running rule Set an integer cgroup setting via sd_bus
Debug: always triggered
Debug: Running effect sd_bus_setting

XXX PRE:
cat: /sys/fs/cgroup/sudo1000.slice/memory.max: No such file or directory
Debug: get_property: path=/org/freedesktop/systemd1/unit/sudo1000_2eslice, interface=org.freedesktop.systemd1.Slice, prop=MemoryMax, type=t
Debug: get_property: type=t, r=1, val=89997312
XXX adaptived_sd_bus_set_ll: First get adaptived_sd_bus_get_ll() target=sudo1000.slice, property=MemoryMax, ll_value2=89997312, ret=0
XXX POST:
cat: /sys/fs/cgroup/sudo1000.slice/memory.max: No such file or directory

Debug: set_property: name=sudo1000.slice, property=MemoryMax
Debug: get_type_from_string: s=sudo1000.slice
Debug: property_assignment: t=0, property=MemoryMax
Debug: update_property: property=MemoryMax, value->value.ll_value=89997312
XXX POST set_property():
/sys/fs/cgroup/sudo1000.slice/memory.max
89997312

Debug: get_property: path=/org/freedesktop/systemd1/unit/sudo1000_2eslice, interface=org.freedesktop.systemd1.Slice, prop=MemoryMax, type=t
Debug: get_property: type=t, r=1, val=89997312
Debug: adaptived_sd_bus_set_ll: validate success.
Debug: adaptived main loop exceeded max loops
Debug: Cleaning up rule Set an integer cgroup setting via sd_bus
Debug: Cleaning up cause always
Debug: Cleaning up effect sd_bus_setting
Debug: stop_transient: sudo /bin/systemctl stop sudo1000.slice > /dev/null 2>&1, ret=0
```
At first /sys/fs/cgroup/sudo1000.slice/memory.max does not exist, but the MemoryMax property does from its prior setting.